Animations/src/components/pulsating-animation.js

```diff
@@ -0,0 +1,82 @@
+import React, {useEffect} from 'react';
+import {Animated, Easing, View, Text} from 'react-native';
+import {blue, lightBlue, pink} from 'stylesheets/colors';
+import {flex1, horizontalCenter, center} from 'stylesheets/global';
+import {py20} from 'stylesheets/spacing';
+import {fs30, bold, uppercase} from 'stylesheets/typography';
+
+const circleSize = 100;
+const animDuration = 5000;
+const minValue = 0.1;
+const maxValue = 0.9;
+
+const PulsatingAnim = () => {
+  const fadeAnim = new Animated.Value(minValue);
+
+  useEffect(() => {
+    Animated.loop(
+      Animated.sequence([
+        Animated.timing(fadeAnim, {
+          toValue: maxValue,
+          duration: animDuration,
+          easing: Easing.circle,
+          useNativeDriver: false,
+        }),
+        Animated.timing(fadeAnim, {
+          toValue: minValue,
+          duration: animDuration,
+          easing: Easing.circle,
+          useNativeDriver: false,
```

Comment:
Same

Reply:
On setting useNativeDriver: true, following error occurred.
`Error: Style property 'backgroundColor' is not supported by native animated module.`